modules/wallet/wallet.go

```diff
@@ -43,6 +43,32 @@ type spendableKey struct {
 	SecretKeys       []crypto.SecretKey
 }
 
+// broadcastedTSet is a helper struct to keep track of transaction sets and to
+// help rebroadcast them.
+type broadcastedTSet struct {
+	height       types.BlockHeight
+	tries        int
+	confirmedTxn map[types.TransactionID]bool
+	transactions []types.Transaction
+}
```

Comment:
should probably move this code into `broadcast.go` or something. I think there's enough logic here that it deserves its own file.